jenkins/bootstrap.py

```diff
@@ -118,20 +137,21 @@ def checkout(repo, branch, pull):
     """Fetch and checkout the repository at the specified branch/pull."""
     if bool(branch) == bool(pull):
         raise ValueError('Must specify one of --branch or --pull')
+
     if pull:
-        ref = pull_ref(pull)
+        refs, checkouts = pull_ref(pull)
     else:
-        ref = branch
+        refs, checkouts = [branch], ['FETCH_HEAD']
 
     git = 'git'
     call([git, 'init', repo])
     os.chdir(repo)
+    call([git, 'config', '--local', 'user.name', 'K8S Bootstrap'])
+    call([git, 'config', '--local', 'user.email', 'k8s_bootstrap@localhost'])
```

Comment:
Should we use a real contact email? Also can you leave a comment about why we need this?